data.sql

```diff
@@ -11,3 +11,39 @@ INSERT INTO animals (name, date_of_birth, weight_kg,neutered, escape_attempts) V
 ('Angemon','2005-06-12',-45.0,true,1),
 ('Boarmon','2005-06-07',20.4,true,7),
 ('Blossom','1998-10-13',17.0,true,3);
+
+INSERT INTO owners(full_name, age) 
+VALUES 
+('Sam Smith',34),
+('Jennifer Orwell',19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
```

Comment:
- [ ] Let us modify our insertion in this way

 INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);

 INSERT INTO owners (full_name, age)
 VALUES ('Jennifer Orwell', 19);

 INSERT INTO owners (full_name, age)
 VALUES ('Bob', 45);

 INSERT INTO owners (full_name, age)
 VALUES ('Melody Pond', 77);

 INSERT INTO owners (full_name, age)
 VALUES ('Dean Winchester', 14);

 INSERT INTO owners (full_name, age)
 VALUES ('Jodie Whittaker', 38);

Reply:
:heavy_check_mark: This is more readable now :+1: